docs/source/contributing.rst

```diff
@@ -0,0 +1,121 @@
+or have suggestions, please refer to the `issue tracker`_ on GitHub.
+
+.. _`issue tracker`: https://github.com/poliastro/poliastro/issues
+
+Documentation
+-------------
+
+Documentation can always be improved and made easier to understand for
+newcomers. The docs are stored in text files under the `docs/source`
+directory, so if you think anything can be improved there please edit the
+files and proceed in the same way as with `code writing`_.
+
+The Python classes and methods also feature inline docs: if you detect
+any inconsistency or opportunity for improvement, you can edit those too.
+
+Besides, the `wiki`_ is open for everybody to edit, so feel free to add
+new content.
+
+To build the docs, you must first create a development environment (see
+below) and then in the ``docs/`` directory run::
+
+    $ cd docs
+    $ make html
+
+After this, the new docs will be inside ``build/html``. You can open
+them by running an HTTP server::
+
+    $ cd build/html
+    $ python -m http.server
+    Serving HTTP on 0.0.0.0 port 8000 ...
+
+And point your browser to http://0.0.0.0:8000.
+
+Code writing
+------------
+
+.. image:: https://img.shields.io/waffle/label/poliastro/poliastro/1%20-%20Ready.svg?style=flat-square
+   :target: https://waffle.io/poliastro/poliastro
+   :alt: 'Stories in Ready'
+
+Code contributions are welcome! If you are looking for a place to start,
+help us fixing bugs in poliastro and check out the `"easy" tag`_. Those
+should be easier to fix than the others and require less knowledge about the
+library.
+
+.. _`"easy" tag`: https://github.com/poliastro/poliastro/issues?q=is%3Aissue+is%3Aopen+label%3Aeasy
+
+If you are hesitant on what IDE or editor to use, just choose one that
+you find comfortable and stick to it while you are learning. People have
+strong opinions on which editor is better so I recommend you to ignore
+the crowd for the time being - again, choose one that you like :)
+
+If you ask me for a recommendation, I would suggest PyCharm (complete
+IDE, free and gratis, RAM-hungry) or vim (powerful editor, very lightweight,
+steep learning curve). Other people use Spyder, emacs, gedit, Notepad++,
+Sublime, Atom...
+
+You will also need to understand how git works. git is a decentralized
+version control system that preserves the history of the software, helps
+tracking changes and allows for multiple versions of the code to exist
+at the same time. If you are new to git and version control, I recommend
+following `the Try Git tutorial`_.
+
+.. _`the Try Git tutorial`: https://try.github.io/
+
+If you already know how all this works and would like to contribute new
+features then that's awesome! Before rushing out though please make sure it
+is within the scope of the library so you don't waste your time -
+`the mailing list`_ or `the chat`_ are good places to ask.
+
+.. _`the mailing list`: https://groups.io/g/poliastro-dev
+.. _`the chat`: https://riot.im/app/#/room/#poliastro:matrix.org
+
```

Comment:
I would prefer the code quality block here! @Juanlu001  will tell more if something is needed. As a lot of contributors get confused in running `black` and `isort`, it will be better to have it documented here. 

Specifically, we would want these two commands: 

```
black changed_filename.py
isort changed_filename.py
```
here.

Reply:
@azlkiniue Actually they're required to pass the tests (had to learn this the hard way). Maybe consider adding a section for configuring those tools on some of the most popular IDEs or link to the corresponding wikis of each.

Reply:
Related: https://github.com/poliastro/poliastro/issues/608 I think we should address this one first!

Reply:
Yes :/